app/views/compute_resources_vms/form/libvirt/_base.html.erb

```diff
@@ -5,9 +5,13 @@
 <%= counter_f f, :cpus, :disabled => !new_vm, :label => _('CPUs'), :label_size => 'col-md-2', :'data-soft-max' => compute_resource.max_cpu_count %>
 
 <%= select_f f, :cpu_mode, Foreman::Model::Libvirt::CPU_MODES, :to_s, :to_s, { }, :label => _("CPU mode"), :label_size => "col-md-2" %>
-
-<%= byte_size_f f, :memory, :disabled => !new_vm, :label => _('Memory'), :label_size => "col-md-2", :'data-soft-max' => compute_resource.max_memory %>
```

Comment:
we could just change the `byte_size_f` implementation to leverage the react [`InputFactory`](https://github.com/theforeman/foreman/blob/0add9c59c0edfe23fe235e34834a50e71198b8fd/webpack/assets/javascripts/react_app/components/common/forms/InputFactory.js#L13) like e.g. https://github.com/theforeman/foreman/blob/0add9c59c0edfe23fe235e34834a50e71198b8fd/app/helpers/form_helper.rb#L415
I'm ok to do it as follow-up and even doing it myself :)